### PR TITLE
Add Managed Service related upgrade checks

### PR DESCRIPTION
**Pull request by fbalak**
Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1074/

File: tests/managed-service/test_ms_upgrade.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+logger = logging.getLogger(name=__file__)
+
+
+@pre_upgrade
+@ms_consumer_required
+def test_prepare_block_md5_before_upgrade(block_md5):
+    """
+    Prepare md5 results for utilized RBD PVC.
+
```
Comment (yitzhak12) [inline tests/managed-service/test_ms_upgrade.py:20]: Maybe I would add here a log message: `logger.info(f"RBD file md5 before upgrade: {block_md5}")`

File: tests/managed-service/test_ms_upgrade.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    """
+    pass
+
+
+@pre_upgrade
+@ms_consumer_required
+def test_prepare_fs_md5_before_upgrade(fs_md5):
+    """
+    Prepare md5 results for utilized Ceph FS PVC.
+
```
Comment (yitzhak12) [inline tests/managed-service/test_ms_upgrade.py:30]: Same here - to add a log message: `logger.info(f"Ceph FS file md5 before upgrade: {fs_md5}")`

**Review by yitzhak12 — COMMENTED** [review]
Except for two small comments, the pr looks good.

**Review by jilju — APPROVED** [review]

**Review by suchita-g — APPROVED** [review]

File: tests/managed-service/test_ms_upgrade.py (hunk 1 above)
Comment (fbalak) [reply tests/managed-service/test_ms_upgrade.py:20]: md5 strings are logged when they are taken in respective fixtures

Comment (yitzhak12) [reply tests/managed-service/test_ms_upgrade.py:20]: Yes, it's not mandatory.

**Review by yitzhak12 — APPROVED** [review]

**Pull request merged**